working/macros/feature-specification.md

```diff
@@ -896,6 +916,28 @@ It is a compile-time error if an macro class constructor invoked by a macro
 application has a parameter whose type is not Code (or any subtype of it) or
 one of the aforementioned primitive types (or a nullable type of any of those).
 
+#### Identifier Scope
+
+The following rules apply to any `Identifier` passed as an argument to a macro
+application, whether as a part of a `Code` expression or directly as an
+`Identifier` instance.
+
+The scope of any `Identifier` argument is the same as the scope in which the
+identifier appears in the source code, which is the same as the scope for any
+other type of annotation on a declaration. This means:
+
+* Identifiers in macro application arguments may only refer to static and top
+  level members.
+* They cannot refer to local or instance variables, as those can never be in scope
+  where a macro application appears.
+* Identifiers referring to static class members may be unqualified if the
```

Comment:
You mention "may be unqualified" here. Does this mean some identifiers *may* be qualified? Do we support:

```dart
@MyMacro(some_prefix.SomeClass)
@MyMacro(SomeClass.someStaticMember)
```

?

Reply:
Yes, both should be supported. I can add a section clarifying how these appear to the macro author (in both cases they would only see the unqualified name).

Reply:
I'll add a third, one with two dots:

`@MyMacro(some_prefix.SomeClass.someStaticMember)`